spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigMapPropertySource.java

```diff
@@ -56,9 +57,9 @@ public ConfigMapPropertySource(KubernetesClient client, String name) {
 	}
 
 	public ConfigMapPropertySource(KubernetesClient client, String name, String namespace,
```

Comment:
I don't see the change back.

Reply:
You don't see the change back because I squashed the changes into a single commit and force pushed to this pull request branch.

Reply:
The PR still shows a new one, so not sure what you did, but this PR doesn't have that change.

Reply:
Got it. I put back all original constructors and added a new constructor.